src/giphy-gifme.coffee

```diff
@@ -17,6 +17,8 @@
 api_key = process.env.HUBOT_GIPHY_API_KEY or 'dc6zaTOxFJmzC' # <== Giphy's public API key, please request your own!
 rating_limit = process.env.HUBOT_GIPHY_RATING or 'pg'
 
+Array::empty = -> @.length == 0
```

Comment:
I suppose it doesn't matter in such a small script, but altering a built-in object's prototype is a bad practice.

I need to do some refactoring to get https://github.com/bencentra/hubot-giphy-gifme/pull/3 up to date and merged in, so I'll clean it up then.